### PR TITLE
#1019

### PR DESCRIPTION
**Pull request by rmandelb**
This pull request covers a few small-ish issues: #1016 #1019 #1020  (I started hoping to do #1019 and then added a few other small ones; sorry it's a grab bag).

### CODE REVIEW

**Comment by rmandelb** [comment]
@rmjarvis - might you be able to take a look at this in the coming few weeks?  I hope it'll be a quick fix for a few issues.

File: examples/demo13.py [hunk 1]
```diff
@@ -352,6 +352,10 @@ def main(argv):
         # First, we include the expected Poisson noise:
         final_image.addNoise(poisson_noise)
 
+        # At this point in the image generation process, an integer number of photons gets
+        # detected, hence we have to round the pixel values to integers:
+        final_image.quantize()
```
Comment (rmjarvis) [inline examples/demo13.py]: I believe at this point, the image is already quantized unless any of the pre-noise values were > 2^30.  That's when our Poisson implementation switches over to the Gaussian approximation, which won't necessarily produce integers.
So probably we don't need this line.  Or if you want to keep it, maybe add some commentary about how it's not really necessary for non-saturated pixel values.

Comment (rmandelb) [reply examples/demo13.py]: Yes - I confirmed this directly as well, so have removed this step and added a comment.

**Comment by rmandelb** [comment]
Thanks for approving, @rmjarvis !  I will  plan to merge on Friday unless anybody comments that they want more time to look this over.  

@beckermr - note that a fix for the bug you reported in #1020 is included in this PR.

**Comment by beckermr** [comment]
Thanks @rmandelb !

**Comment by rmandelb** [comment]
Merging #1022.

**Pull request merged**